cmake/finders/FindLibAJANTV2.cmake

```diff
@@ -56,11 +56,30 @@ if(PKG_CONFIG_FOUND)
 endif()
 
 find_path(
-  LibAJANTV2_INCLUDE_DIR
-  NAMES ajalibraries
+  _libajantv2_new_include_dir
```

Comment:
The result of `find_path` writes into a function with directory scope - directory scope variables have to be all uppercase.